tests/v2/validation/certrotation/cert_rotation.go

```diff
@@ -337,7 +339,7 @@ func getCertificatesFromV3Node(client *rancher.Client, v3Node *management.Node)
 	return certificates, nil
 }
 
-func downloadRKE1SshKeys(client *rancher.Client, v3Node *management.Node) ([]byte, error) {
+func downloadRKE1SSHKeys(client *rancher.Client, v3Node *management.Node) ([]byte, error) {
```

Comment:
Would there be a way to accomplish this with V1 instead of V3 objects?

Reply:
this was just an update to the naming, not to the func. But perhaps, I can look into it. Can this be a separate issue from this PR?

Reply:
Sure. No need to scope creep this PR.